src/System.EventLog/src/System/Diagnostics/SharedUtils.cs

```diff
@@ -0,0 +1,335 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Security.AccessControl;
+using System.Security.Permissions;
+using System.Security.Principal;
+using System.Text;
+using System.Threading;
+using Microsoft.Win32;
+using Microsoft.Win32.SafeHandles;
+
+namespace System.Diagnostics
+{
+    internal static class SharedUtils
+    {
+
```

Comment:
nit, extra line. You can use regexes to find this kind of thing if you like, eg.,
`{ *\n *\n` will find open braces with ablank line after and you can replace with `{\n`

Reply:
ohky